system-test/dns.js

```diff
@@ -142,16 +135,6 @@ var DNS_DOMAIN = process.env.GCLOUD_TESTS_DNS_DOMAIN;
     ZONE.delete({ force: true }, done);
   });
 
-  it('should create a zone', function(done) {
```

Comment:
I don't think so. I think only because we were creating all over the place, and we usually just use the fact that it was created in the before hook to prove it works.